.babelrc

```diff
@@ -1,12 +1,7 @@
 {
-  "presets": [["airbnb", {
-    "targets": {
-      "node": 6
-    }
-  }]],
+  "presets": ["@babel/preset-react", "@babel/preset-env"],
   "plugins": [
-    "add-module-exports",
-    "transform-flow-strip-types",
-    "transform-class-properties"
+    "@babel/plugin-transform-flow-strip-types",
+    "@babel/plugin-proposal-class-properties"
   ]
 }
```

Comment:
@paradoxxxzero Any reason you removed the `airbnb` preset and `add-module-exports` plugin?